### ISSUE TITLE
creating stores springcloud/stores times out

### ISSUE THREAD

**Issue by hpejcinovic-pivotal** [spring-cloud-samples/customers-stores#17]
when running 
ltc create --memory-mb 0 --timeout 5m -e MONGODB_HOST=10.0.1.170 -e MONGODB_PORT=61002 -e RABBITMQ_HOST=10.0.1.170 -e RABBITMQ_PORT=61001 stores springcloud/stores

it times out

Timed out waiting for the container to come up.
This typically happens because docker layers can take time to download.
Lattice is still downloading your application in the background.
To view logs:
	ltc logs stores
To view status:
	ltc status stores

App will be reachable at:
http://stores.192.168.11.11.xip.io

First springcloud/customers downloads and runs fine

**Comment by spencergibb** [comment]
Not sure what we can do about this.

**Comment by hpejcinovic-pivotal** [comment]
Fair enough. I've tried it few times from scratch, deleting it with ltc remove stores, but it always times out.
